CHANGELOG.md

```diff
@@ -4,6 +4,95 @@
+in addition to the default bottom placement. The placement is persisted in the Prompt Cell
+Position setting.
+
+The console toolbar now includes buttons to:
+- execute code,
+- restart the kernel,
+- clear the cells,
+- switch the kernel.
+
+Additional settings for console behavior were added:
+- Clear Code Content on Execute (enabled by default) - when disabled,
+  the code submitted for execution remains in the code editor, allowing for further edits
+- Hide Code Input - when enabled shows only the output of the execution in the console
+- Clear Cells on Execute - when enabled shows only last cell
+
+Toggling all the three new settings transforms the console into an interactive editor
+resembling an ephemeral notebook with a single cell.
+
+<img alt="The option to change console prompt position is available under the three dots button in the console toolbar, which is the last toolbar button" src="https://raw.githubusercontent.com/krassowski/jupyterlab/user-facing-changelog-4.4/docs/source/getting_started/changelog_assets/4.4-console.png" class="jp-screenshot">
+
+### Settings import and export
+
+Settings can now be exported to `overrides.json` from the Settings Editor,
+which can be used to pre-configure defaults on deployments or for restoring
+the settings.
+
+<img alt="The Import and Export buttons in the Settings Editor are the first two items in the toolbar panel" src="https://raw.githubusercontent.com/krassowski/jupyterlab/user-facing-changelog-4.4/docs/source/getting_started/changelog_assets/4.4-settings-import-export.png" class="jp-screenshot">
+
+### Workspace indicator
+
+An opt-in workspace indicator displayed on the top bar (next to the menu bar) can be enabled
+in the Settings → View → Appearance → Show Workspace Indicator.
+
+<img alt="The workspace indicator in the open state" src="https://raw.githubusercontent.com/krassowski/jupyterlab/user-facing-changelog-4.4/docs/source/getting_started/changelog_assets/4.4-workspace-indicator.png" class="jp-screenshot">
+
+### Support for collaboration without `RTC:` drive
+
+Real time collaboration in previous versions of JupyterLab
+(which can be enabled by installing jupyter-collaboration package)
+was implemented by swapping the default JupyterLab file system
+drive to a collaborative drive with the `RTC:` prefix.
+This was not compatible with multiple extensions.
+JupyterLab 4.4 introduces Content Providers API which will
+enable the next version of `jupyter-collaboration` to work
+without changing the drive prefix, and thus improves
+compatibility with extensions which do not support arbitrary drives.
+
+### Kernel subshells support
+
+Kernel subshells, introduced in [JEP 91](https://jupyter.org/enhancement-proposals/91-kernel-subshells/kernel-subshells.html),
+enable concurrent code execution, in kernels which support them.
+When performing a long-running computation (such as training of a model), subshells enable:
```

Comment:
```suggestion
When performing long-running computations (such as training a model), subshells allow users to:
```

Reply:
Took everything except for enable → allow change to follow the docs writing guide.